src/AppInstallerCommonCore/AppInstallerStrings.cpp

```diff
@@ -137,6 +137,12 @@ namespace AppInstaller::Utility
         return result;
     }
 
+    std::u32string ConvertToUTF32(std::string_view input)
+    {
+        std::wstring_convert<std::codecvt_utf8<char32_t>, char32_t> convU32;

```

Comment:
>std::wstring_convert<std::codecvt_utf8<char32_t>, char32_t> [](start = 8, length = 59)

This is deprecated in C++17.  You will either need to find another way to convert to UTF-32, or use the break iterator rather than converting the entire string.